benefit/data/sheets/WorkBasketResultFields/WorkBasketResultFields.json

```diff
@@ -9,19 +9,28 @@
   {"LiveFrom": "01/01/2018", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "interlocReviewState", "Label": "Interlocutory review state", "DisplayOrder": 8},
   {"LiveFrom": "25/09/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "linkedCasesBoolean", "Label": "Has related appeal(s)?", "DisplayOrder": 9},
   {"LiveFrom": "25/09/2019", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "[LAST_STATE_MODIFIED_DATE]", "Label": "Date of state change", "DisplayOrder": 10, "ResultsOrdering": "1:ASC"},
+  {"LiveFrom": "25/05/2022", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "dwpIsOfficerAttending", "Label": "Is the Presenting Officer attending?", "DisplayOrder": 11},
+  {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "workBasketHearingDate", "Label": "Hearing Date", "DisplayOrder": 18},
+  {"LiveFrom": "19/05/2022", "CaseTypeID": "Benefit", "UserRole": "caseworker-sscs-clerk", "CaseFieldID": "workBasketHearingEpimsId", "Label": "Hearing Venue", "DisplayOrder": 19},
```

Comment:
Same again, display order doesn't look right

Reply:
Will leave as this was tested this way and passed SIT.